### PR TITLE
 Randomize Images Pulled from MET API

### PR DESCRIPTION
**Pull request by FatimahAreola**
This PR is related to the story "Randomize Images Pulled from MET API"

Tasks
- Identify and store artwork that could be candidates for use in the game
- Generate selection of artwork based on criteria at initialization of new game

Testing
Are you able to select one of the 8 artists listed in the game and play an iteration of the game exclusively with art from this artist? (these are 8 different tests)
If you do not select a valid artist, are you met with a randomized set of images?

This PR also includes a test document that outlines the following unit tests/edge cases (5 in total):
Missing Artist Value - empty string, null value, and no string assignment at all
Invalid Artist Value - artist that does not exist in system, integer value instead of string

### CODE REVIEW

**Comment by FatimahAreola** [comment]
Updated from Main since I saw that there were conflicts :)

File: server/app.py [hunk 1]
```diff
@@ -111,18 +111,50 @@ def fetchArtInformation(i):
...
+    [437397,437392,337491,437390,437402,437396,437389,437398],
+    [10793,10786,10795,10796,10794,10790,10788,10798],
+    [10154,10155,10151,10150,10158,21186,10156,10149],
+    [435882,435885,437990,435871,435874,435873,435870,435879],
+    [786115,370826,365307,337070,786093,785991,785974,691010],
+    [752047,11865,11859,11862,11866,11860,11863,11872]]
+    
+    artists= ["Paul Gauguin", "Vincent van Gogh", "Rembrandt", "Asher Brown Durand", "Albert Bierstadt", "Paul Cézanne", "Auguste Edouart", "Frederic Remington"]
+    
+    for a in range(8):
```
Comment (CarlaLeal) [inline server/app.py]: Instead of iterating through the array, maybe you could use python's index function to your advantage?
`artistIdx = artists.index(selectedArtist)`

Comment (FatimahAreola) [reply server/app.py]: Great suggestion! Wish I had thought of that - I've implemented it!

**Comment by FatimahAreola** [comment]
Resolved conflicts and merged from main

File: server/app.py (hunk 1 above)
Comment (FatimahAreola) [reply server/app.py]: Now that I am testing, I remember that we do need to iterate through the array to ensure that the selected artist does in fact exist in the array. I thought the index function would return a -1 value for example, but it throws an error so I am not sure it is the best implementation

**Review by CarlaLeal — CHANGES_REQUESTED** [review]
When the artistDisplayName does not exist, the card's artistName property does returns null, which then affects the display of artists in the game. Could we introduce a check to see if it exists before assigning the value of the property?

**Comment by FatimahAreola** [comment]
Hey @CarlaLeal when I made the changes you requested, I believe it broke some of the frontend that Abi wrote because she has her code set up to pass values even if they are blank. (We had a discussion about it on one of the PRs this sprint, here is the thread: https://github.com/FatimahAreola/MSCI-342-Group-Project/pull/25)

Based on our discussion of why you're requesting this change, I believe it's because the favorite artists feature looks for the string of all artists included in the game, and if a blank string is passed it creates the option of a blank artist. Since this is a frequent issue with the MET API, may I recommend an alternate solution that would be to check if an artist name is blank before offering it to the user as an option of an artist to favorite? The generate game based on artists function in the app already accommodates for if it is passed a blank string so this shouldn't be a concern for this aspect.

Let me know what you think! I'm open to troubleshooting through the design suggestion you made if you think it's better than my proposal. My take on the matter is that it is likely preferable not to break other components of the system (like Abi's code).

**Comment by FatimahAreola** [comment]
Hi @CarlaLeal, I sent some messages over Slack explaining my reasoning for implementing the changes you asked for in a different branch. I believe the current state of this PR works and am walking through some testing with Mo right now. Here is the link to the separate branch with the changes you requested: https://github.com/FatimahAreola/MSCI-342-Group-Project/tree/Artist-Name-Fix

**Comment by m2qaderi** [comment]
Hi Fatimah! I'm so glad we were able to get on a call and test this branch together. As you know, everything looks great on my end, just make those small updates we discussed!

**Review by m2qaderi — APPROVED** [review]

**Comment by m2qaderi** [comment]
@CarlaLeal since the changes you've requested have been moved to a new branch, I am going to go ahead and merge this
